content/docs/rfcs/61/README.md

```diff
@@ -0,0 +1,373 @@
+---
+slug: 61
+title: 61/STATUS-Community-History-Archives
+name: Status Community History Archives
+status: raw
+category: Informational
+editor: r4bbit <r4bbit@status.im>
+contributors:
+  - Sanaz Taheri <sanaz@status.im>
+  - John Lea <john@status.im>
+---
+
+# Abstract
+
+Messages are stored permanently by store nodes ([13/WAKU2-STORE](/spec/13/)) for up to a certain configurable period of time, limited by the overall storage provided by a store node.
+Messages older than that period are no longer provided by store nodes, making it impossible for other nodes to request historical messages that go beyond that time range. 
+This raises issues in the case of Status communities, where recently joined members of a community are not able to request complete message histories of the community channels.
+
+This specification describes how **Control Nodes** archive historical message data of their communities, beyond the time range limit provided by Store Nodes using the [BitTorrent](https://bittorrent.org) protocol.
+It also describes how the archives are distributed to community members via the Status network, so they can fetch them and gain access to a complete message history.
+
+# Terminology
+
+The following terminology is used throughout this specification:
+
+| Name                 | References |
+| -------------------- | --- |
+| Waku node            | An Waku node ([10/WAKU2](/spec/10/)) that implements [11/WAKU2-RELAY](/spec/11/)|
+| Store node           | A Waku node that implements [13/WAKU2-STORE](/spec/13/) |
+| Waku network         | A group of Waku nodes forming a graph, connected via [11/WAKU2-RELAY](/spec/11/) |
+| Status user          | An Status account that is used in a Status consumer product, such as Status Mobile or Status Desktop |
+| Status node          | A Status client run by a Status application |
+| Control node      | A Status node that owns the private key for a Status community |
```

Comment:
At this point in time, there can theoretically multiple control nodes, if you share the private key for the community with another node (owning private key == being a control node).

However, this is currently being changed, such that, one can securely transfer a private key to another node, but it will then be removed from the current node, resulting in only one node actively maintaining the key.